man/udev.xml

```diff
@@ -525,6 +525,9 @@
             <term><varname>GOTO</varname></term>
             <listitem>
               <para>Jumps to the next <varname>LABEL</varname> with a matching name.</para>
+
+              <para>Multiple <varname>GOTO</varname> assignments in a single
+              rule are not supported.</para>
```

Comment:
"not supported" does not really mean the same as "discarded". Maybe be explicit and say "Only the first GOTO is used. Any later ones are ignored after a warning.", but worded in a better way.

Reply:
> "not supported" does not really mean the same as "discarded". Maybe be explicit and say "Only the first GOTO is used. Any later ones are ignored after a warning.", but worded in a better way.

The original wording was the following:
```
In each rule only the first GOTO key takes effect, all the rest are discarded.
```
However, it was rejected in the earlier review https://github.com/systemd/systemd/pull/26778#pullrequestreview-1336994402

Now there are two opposing reviews for the same little piece of documentation.